datafusion/physical-plan/src/lib.rs

```diff
@@ -242,45 +242,6 @@ pub fn with_new_children_if_necessary(
 /// Return a [wrapper](DisplayableExecutionPlan) around an
 /// [`ExecutionPlan`] which can be displayed in various easier to
 /// understand ways.
-///
-/// ```
-/// use datafusion::prelude::*;
```

Comment:
I'm guessing this has been removed because the frontend is not a dependency of physical-plan. Perhaps we could manually construct the physical plan? Don't feel strongly